.github/workflows/ci.yml

```diff
@@ -0,0 +1,194 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - uses: actions/setup-node@v2-beta
+      with:
+        node-version: '10.x'
```

Comment:
If you prefer, we can create an environment variable `NODE_VERSION` that is scoped to the entire workflow:

https://gist.github.com/ijlee2/25004d700cd008ee08ecc4873c109a5b#file-ci-yml-L53

Reply:
Good catch! This would simplify upgrading the node version later. :+1:

Created https://github.com/jelhan/create-github-actions-setup-for-ember-addon/issues/14 to track it.